components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -325,13 +325,13 @@ PublisherInfoDatabase::GetPublisherInfo(const std::string& publisher_key) {
 
 ledger::PublisherInfoPtr
 PublisherInfoDatabase::GetPanelPublisher(
-    const ledger::ActivityInfoFilter& filter) {
+    ledger::ActivityInfoFilterPtr filter) {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
 
   bool initialized = Init();
   DCHECK(initialized);
 
-  if (!initialized || filter.id.empty()) {
+  if (!initialized || filter->id.empty()) {
```

Comment:
We should check if filter is null here first

Reply:
fixed